typescript/quicksight/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "quicksight-example",
+  "version": "0.1.0",
+  "bin": {
+    "quicksight-example": "bin/quicksight-example.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
```

Comment:
No test in app. Need to remove this otherwise the test step in the build will fail.